iModelCore/GeomLibs/PublicAPI/Geom/CurveDetails.h

```diff
@@ -515,7 +515,15 @@ double m_arcAngle;  // (if this is positive) turns larger than this become arcs.
 double m_chamferAngle; // (if this is positive) "outer chamfers" are created with this max angle.
 double m_offsetDistance;
 bool   m_forceClosure;
-bool   m_unusedBool[8];
+bool m_allowSharpestCorners; // Whether to remove the internal turn angle upper bound for sharp corner construction.
+                             // By default, a sharp corner is not created at a joint when the turn angle is too large,
+                             // so as to avoid offsets whose ranges blow up. Internally, this is implemented by applying an
+                             // upper bound to `m_chamferAngle`.
+                             // When `m_allowSharpestCorners` is true, this internal upper bound is removed, allowing sharp
+                             // corners for turn angles up to `m_chamferAngle`. Thus, if you know your input turn angles
+                             // are no greater than `m_chamferAngle`, you can create an offset with sharp corners at
+                             // each joint by setting `m_chamferAngle < m_arcAngle` and `m_allowSharpestCorners` to true.

```

Comment:
This makes clear user is setting the arc angle, not the chamfer angle, which is already known.

```suggestion
                             // each joint by setting `m_arcAngle > m_chamferAngle` and `m_allowSharpestCorners` to true.

```